src/middlewares/userMiddlewares.js

```diff
@@ -37,14 +38,33 @@ export default class UserMiddlewares {
   static async checkUserExistBeforeLogin(req, res, next) {
     try {
       const { email } = req.body;
-      const data = await userServices.getUserByEmail(email);
+      const data = await UserServices.getUserByEmail(email);
       if (data) {
         req.user = data.dataValues;
         return next();
       }
-      return resError(res, 404, 'User does not exist.');
+      return resError(res, 404, 'Unsuccesful, User does not exist, kindly check to confirm email is correct.');
```

Comment:
Please return this error message to what it was before

Reply:
ok